packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/create-database/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "pci_databases_databases_create_database_confirm": "Creer la base de données",
+  "pci_databases_databases_create_database_cancel": "Annuler",
+  "pci_databases_databases_create_database": "Créer une base de donnée",
+  "pci_databases_databases_create_database_warning": "Attention, c'est très dangereux parceque pas branché à l'api",
```

Comment:
Not the definitive content, right ?

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1